aa_stripe/api.py

```diff
@@ -52,7 +54,13 @@ def post(self, request, *args, **kwargs):
             return Response(status=400, data={"message": "already received"})
         except StripeWebhook.DoesNotExist:
             # correct, first time. Create webhook
-            webhook = StripeWebhook.objects.create(id=event["id"], raw_data=data["raw_data"])
+            try:
+                webhook = StripeWebhook.objects.create(id=event["id"], raw_data=data["raw_data"])
+            except stripe.error.InvalidRequestError:
```

Comment:
webhooks are not just for coupons. 
What if this will be subscription? Message will be "coupon with this coupon_id does not exists at Stripe API" so wrong

Reply:
I added a field "parse_error" to store error information, webhooks api should always return the StripeWebhook object on success (even when parsing failed).